pyproject.toml

```diff
@@ -132,6 +132,17 @@ requires-python = '>=3.9'
 'process.workflow.workchain' = 'aiida.orm.nodes.process.workflow.workchain:WorkChainNode'
 'process.workflow.workfunction' = 'aiida.orm.nodes.process.workflow.workfunction:WorkFunctionNode'
 
+[project.entry-points.'aiida.orm']
```

Comment:
@sphuber correct me if I'm wrong, but this has to do with the use of `orm_class`, right? Under the [implementation](https://github.com/aiidateam/AEP/blob/983a645c9285ba65c7cf07fe6064c23e7e994c06/010_orm_schema/readme.md#implementation) section of the AEP, it states that `orm_class` is a shortcut to `model_to_orm` used to define a model field that points to an ORM instance. So instead of
```python
class Model(...):
    user = MetadataField(
        ...,
        model_to_orm=lambda pk: User.collection.get(id=pk),
    )
```
you can do
```python
class Model(...):
    user = MetadataField(
        ...,
        orm_class=User,
    )
```
The various ORM classes that may be used in `orm_class` are then defined in `project.entry-points.'aiida.orm'`, so that you can do in `Entity.model_to_orm_field_values(...)`:
```python
if orm_class := get_metadata(field, 'orm_class'):
    if isinstance(orm_class, str):
        orm_class = BaseFactory('aiida.orm', orm_class)
        fields[key] = orm_class.collection.get(id=field_value)
```
But then I guess my question is, do these not already have entry points? Or is the problem that you can't fetch them all generically using the `BaseFactory`?